workers/unity/Packages/com.improbable.gdk.core/ReactiveComponents/Systems/ReactiveComponentSendSystem.cs

```diff
@@ -36,17 +40,58 @@ protected override void OnUpdate()
                 return;
             }
 
-            var commandSystem = World.GetExistingManager<CommandSystem>();
+            ReplicateEvents();
+            ReplicateCommands();
```

Comment:
Is it worth grouping the gathering jobs from these together and waiting once for them to complete? 

I.e. - 

`var chunks = GatherChunks();`
`ReplicateEvents(ref chunks);`
`ReplicateCommands(ref chunks);`

Or is the extra bookkeeping not worth it?

Reply:
Given the change that we now check each chunk for the relevant components we could also try things like getting all chunks with a spatailos entity ID in them. I don't know if that would actually be good for everything, but for command senders that might actually be better.